### PR TITLE
Create CONTRIBUTOR_DETAILS

### PR DESCRIPTION
**Pull request by adroition**
Explains itself when you read it.

### CODE REVIEW

File: CONTRIBUTOR_DETAILS [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Contributor Details #
+The following people are the official contributors of the Wolf Freedom Core.
+These people have write access.
+rovertdude
+PuggyCrew
+_c0ded
+Wolfie_YT
```
Comment (rovertdude) [inline CONTRIBUTOR_DETAILS]: Wolfie has no access to the WFC.

**Comment by rovertdude** [comment]
Change the one line and this will be merged.

**Comment by adroition** [comment]
@rovertdude Resolved it

**Comment by rovertdude** [comment]
Thanks for the PR!

**Comment by xOverloaded** [comment]
@adroition I rec you for dev.

**Pull request merged**